internal/fsverity/fsverity_test.go

```diff
@@ -143,8 +143,8 @@ func resolveDevicePath(path string) (_ string, e error) {
 	}
 
 	// resolve to device path
-	maj := (stat.Dev >> 8) & 0xff
-	min := stat.Dev & 0xff
+	major := uint32((stat.Dev&0x00000000000fff00)>>8) | uint32((stat.Dev&0xfffff00000000000)>>32)
+	minor := uint32((stat.Dev&0x00000000000000ff)>>0) | uint32((stat.Dev&0x00000ffffff00000)>>12)
```

Comment:
```suggestion
	maj, min := unix.Major(stat.Dev), unix.Minor(stat.Dev)
```

Reply:
`min` conflicts with built-in Go identifier, full names are also more readable for me. Are you sure it's better to use `maj/min` than `major/minor`?

Reply:
major, minor should be better then

Reply:
Ok, will fix now.
Also, can we skip CI for this change? I don't think we need several dozen VMs for this.

Reply:
No, we should have CI to make sure this does not break our supported platforms.